Lib/http/server.py

```diff
@@ -105,6 +106,12 @@
 import time
 import urllib.parse
 
+try:
+    import ssl
+except ImportError:
+    ssl = None
+
+from getpass import getpass
```

Comment:
Similarly, only import this at time of use to keep import time low for common use cases.